.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] Please, add `.gitignore` file with required file inside (test.md should be also ignored).